### PR TITLE
[feat] basic layout of the webpage

### PR DESCRIPTION
**Pull request by saitunguturu**
[//]: # "Feel free to customize this template to your project's vibes"

### Description

- replaced image url with image
- updated lint:check script to allow for commits
- added title for the page
- added a basic layout for each comment, using separate username and date styles
- lines up the heart icon, # of likes, and share icon but used a space between flex-adjustment

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- can be reviewed all at once
- check the layout.tsx file to see all the new styles made for the text
- based off that check the page.tsx file to see if I implemented them properly with appropriate html tags

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- I added BP logo next to the title, but it is not loading. needs to be fixed.
- need to create the gray outlines for each post & comment

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @kylezryr

### CODE REVIEW

File: app/styles.module.css [hunk 1]
```diff
@@ -7,10 +8,105 @@
   justify-content: center;
   margin: 0;
   padding: 0;
+
+  color: #262626;
+  font-family: Poppins;
```
Comment (kylezryr) [inline app/styles.module.css]: if you check the `layout.tsx` file, you can see that we actually set the 'Poppins' font to the `body` of the app, which applies it to all HTML tags inside our app. we do this because Poppins isnt a CSS default font and needs to be imported, which is why it comes out as times new roman in your app! so you dont actually need to set font-family to Poppins here, its already done!

File: app/page.tsx [hunk 2]
```diff
@@ -8,9 +8,16 @@ export default function Home() {
   return (
     <main className={styles.main}>
       <div className={styles.content}>
+        <h1 className={styles.header}>
+          <img src="assets/images/bp-logo.png" />
+          <span className="color-blue">blueprint</span> blog
+        </h1>
```
Comment (kylezryr) [inline app/page.tsx:14]: my mistake, you didn't actually need to implement the header for the web project (it was just for mobile)! but ++ for trying, it looks good!

File: app/page.tsx [hunk 3]
```diff
@@ -8,9 +8,16 @@ export default function Home() {
...
+
         <ProfileIcon size={24} />
-        <p>rbeggs</p>
-        <p>September 19</p>
+        <div className={styles.headerRow}>
+          <p className={styles.username}>rbeggs</p>
+          <p className={styles.date}>September 19</p>
+        </div>
```
Comment (kylezryr) [inline app/page.tsx:22]: image
 the profile icon, name, and date should all be in one row, and the name+picture should be left aligned and the date is to the right. you'll probably need to define new divs and apply alignments to those divs to achieve this!

File: app/styles.module.css [hunk 4]
```diff
@@ -7,10 +8,105 @@
...
+  flex-direction: column;
+  margin-block: 1.5rem;
+}
+
+.image {
+  /*main image*/
+  width: 21.625rem;
+  height: 14.4375rem;
+  flex-shrink: 0;
+}
```
Comment (kylezryr) [inline app/styles.module.css:44]: the image border should be rounded off as shown below! you can achieve this with `border-radius`

image

File: app/page.tsx [hunk 5]
```diff
@@ -22,27 +29,41 @@ export default function Home() {
           https://www.codetenderloin.org/
         </p>
```
Comment (kylezryr) [inline app/page.tsx:32]: we need some spacing before and after this block of text, consider adding a `margin` to this text!

File: app/page.tsx [hunk 6]
```diff
@@ -22,27 +29,41 @@ export default function Home() {
...
+        />
 
-        <HeartIcon size={24} />
-        <p>256 Likes</p>
-        <ShareIcon size={24} />
+        <div className={styles.iconRow}>
+          <HeartIcon size={24} />
+          <p className={styles.username}>256 Likes</p>
+          <ShareIcon size={24} />
+        </div>
```
Comment (kylezryr) [inline app/page.tsx:45]: you'll need to adjust this div to make the likes count and the heart icon aligned to the left, and align the share icon to the right, which you can still achieve with a space-between, you just need to put the heart icon and likes count into a separate div!

File: app/page.tsx [hunk 7]
```diff
@@ -22,27 +29,41 @@ export default function Home() {
...
+            <div className={styles.commentHeader}>
+              <p className={styles.username}>daviddd</p>
+              <p className={styles.date}>September 20</p>
+            </div>
+            <p>
+              This organization is doing amazing work tackling the complex root
+              causes of the issue.
+            </p>
+          </div>
+        </div>
```
Comment (kylezryr) [inline app/page.tsx:59]: you have the right idea with this div, but the profile icon and the name should be next to each other, instead of the profile icon being next to the whole comment! you'll need to adjust where each component/HTML tag is within the divs to achieve this

image
vs. 
image

File: app/page.tsx [hunk 8]
```diff
@@ -22,27 +29,41 @@ export default function Home() {
...
+        <div className={styles.comment}>
+          <ProfileIcon size={24} className={styles.profileIcon} />
+          <div className={styles.commentContent}>
+            <div className={styles.commentHeader}>
+              <p className={styles.username}>vppraggie</p>
+              <p className={styles.date}>September 21</p>
+            </div>
+            <p>Thanks for sharing!</p>
+          </div>
+        </div>
```
Comment (kylezryr) [inline app/page.tsx:70]: we also need more spacing between each comment and the image itself, add some margins or padding!

**Review by kylezryr — CHANGES_REQUESTED** [review]
great work Sai! a couple of changes to make to make the page reflect the Figma more closely but this is a great start!
